package-testing/sdk-test-runner/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "sdk-test-runner",
+  "version": "1.0.0",
```

Comment:
So the API server you started at `0.1.0` (I think), but this one is `1.0.0` 😅  We beta or not?!

Reply:
you beta believe that we're starting them all at `1.0.0`

Reply:
(I updated the other servers' versions)